starter-code/index.html

```diff
@@ -2,30 +2,65 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  Instagram
+  <div class="fullPage">
 
-  Capture and Share the World's Moments
+  <div class ="topHalf">
+ 
+  <!-- placeholder for the blue color -->
+    
+  </div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+  <div class = "bottomHalf">
+    
+    <img id="phones" src="./images/phones.png" alt="Instgram App on iPhone">
+    
+    <div class="rightSide ">
+    
+        <div class="topInfo"> 
+          <img class="logo" src=./images/brand.png alt="Instagram">
+          <button name="logIn"><img src="./images/home.png" alt="home logo">Log In</button>
+        </div>
+    
+        <div class="textBox">
 
-  Oh yeah, did we mention it's free?
+          <h1>Capture and Share <br>the World's Moments</h1>
+          
+          <p>Instagram is a <strong>fast</strong>, <strong>beautiful</strong> and <strong>fun</strong> way to share your life with friends and family.</p>
+          
+          <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
+          
+          <p>Oh yeah, did we mention it's free?</p>
 
-  Download on the App Store
-  Get it on Google Play
+          <br>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
+          <img class="badge" src=./images/badge-iphone.png alt="Download on the App Store">
+          <img class="badge" src=./images/badge-android.png alt="Get it on Google Play">
+        
+        </div>
+    </div>
+  </div>  
+  
+  <br>
 
-  &copy; 2014 Instagram
+  <footer>
```

Comment:
these links should be slightly darker blue to stand out more and the footer should be a little higher up on the page so as to not be so close to the bottom of the page.